src/main/java/games/strategy/engine/framework/ui/GameChooser.java

```diff
@@ -99,7 +100,10 @@ private void layoutCoponents() {
 
   public static GameChooserEntry chooseGame(final Frame parent, final String defaultGameName)
       throws InterruptedException {
-    final GameChooser chooser = new GameChooser(parent, GameChooserModel.newInstance());
+    final GameChooserModel gameChooserModel = new GameChooserModel(BackgroundTaskRunner.runInBackgroundAndReturn(
```

Comment:
Not directly related to this PR, but this change brings us a step further to "fix the issue" noted in https://github.com/triplea-game/triplea/pull/2572/commits/5f82d972cca463dfba877d1b0cb10a9dbf7c2ac2
In other words: It would be nice If we'd pass a reference to the underlying GameChooserModel object to  the caller.